intake/catalog/tests/test_local.py

```diff
@@ -82,7 +82,7 @@ def test_source_plugin_config(catalog1):
 
 def test_metadata(catalog1):
     assert hasattr(catalog1, 'metadata')
-    assert catalog1['metadata']['test'] is True
```

Comment:
Ah, so the point was that "metadata" was always supposed to be an attribute only...